merkle/crdt/merklecrdt_test.go

```diff
@@ -40,6 +40,7 @@ func newTestBaseMerkleCRDT() (*baseMerkleCRDT, datastore.DSReaderWriter) {
 }
 
 func TestMerkleCRDTPublish(t *testing.T) {
+	t.Parallel()
```

Comment:
nitpick: This line very much appears to be useless, as it is the only such call within the package

Reply:
resolved